.vscode/extensions.json

```diff
@@ -0,0 +1,13 @@
+{
+  "recommendations": [
+    "Nixon.env-cmd-file-syntax",
```

Comment:
Massive nitpick but this one seems non-essential?

Reply:
Agreed, dropping